index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <!-- CSS Stylesheet-->
+        <link rel="stylesheet" type="text/css" href="src/css/style.css">
+        <!--Google fonts-->
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;600;700&display=swap" rel="stylesheet">
+        <!-- Font Awesome icons -->
+        <script src="https://kit.fontawesome.com/0e63fe95a0.js" crossorigin="anonymous"></script>
+        <title>Anghelo's Portfolio</title>
+    </head>
+    <body>
```

Comment:
Let's stick to the design as much as possible. Take a look at the image below. In keeping with the template by the right
- [ ] Please remove the amber-colored margin around your navbar

- [ ] There seems to be some kind of margin around your headline section (as evidenced in your design) that makes your design look different from what is on the template. Please modify this to look more like what is on the template

- [ ] The gray circle on the left of your headline section makes the initial text of your paragraph somewhat unreadable since they have similar colors. Please move the circle a little bit upwards (as we have in the template) so that it does not interfere with your text.

- [ ] Please add some margin (or padding) between the different lists under the **Skills**, **Framework** and **Languages** categories as we have in the template

- [ ] **[OPTIONAL]** It would be nice to also add the smaller right circle as we have in the template

![anghelo-jpg](https://user-images.githubusercontent.com/36542986/138371930-9af4eb04-8926-4f70-b231-eae16cc02927.png)